reflect-lint/build.gradle

```diff
@@ -0,0 +1,20 @@
+apply plugin: 'java-library'
+
+targetCompatibility = JavaVersion.VERSION_1_8
+sourceCompatibility = JavaVersion.VERSION_1_8
+
+dependencies {
+  compileOnly deps.lint.api
+  compileOnly deps.lint.checks
+
+  testImplementation deps.lint.core
```

Comment:
what's `core` needed for?

Reply:
Oh yeah, actually that's not necessary...